portfolio/src/main/webapp/script.js

```diff
@@ -12,17 +12,38 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-/**
- * Adds a random greeting to the page.
- */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+function addRandomProject() {
```

Comment:
It's good practice and part of the JS style guide at Google to add [JSDoc](https://google.github.io/styleguide/jsguide.html#jsdoc) for each function. Could you add a general description of what this function is doing?